src/selectors/entities/channels.js

```diff
@@ -198,7 +259,7 @@ export const getChannelsNameMapInCurrentTeam = createSelector(
 );
 
 // Returns both DMs and GMs
-export const getDirectChannels = createSelector(
+export const getAllDirectChannels = createSelector(
```

Comment:
Agreed, we're working on adding support on the mobile side as well

Reply:
I don't see how this would block Redux PRs.

If you look closely getDirectChannels is still a function.  It's just a different implementation than previous.

If I'm not mistaken test would break if `getDirectChannels` was not included in the selector.


You can take a look at `getDirectChannelIds`.

I think what's confusing is the order in which the functions are implemented.

It was hard to navigate at first when implementing this feature.  Since there's multiple selectors for different type of channels.
So what I did, was to reorganize the order of the functions which makes the git diff look like it's renaming functions